tracks/c/exercises/isogram/mentoring.md

```diff
@@ -11,7 +11,7 @@ A solution with a single loop that uses a lookup table to keep track of letters
 bool is_isogram(const char phrase[])
 {
   size_t phraseLength = strlen(phrase);
-  uint32_t used[26] = { 0 };
+  bool used[26] = { false };
```

Comment:
I realize that C99 has `_Bool` and that standard libraries use `bool`.

How great that we teach so modern C that even old C programmers have something to learn.